product_company_id/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,3 @@
+Sylvain LE GAL <https://twitter.com/legalsylvain>
```

Comment:
This module comes from a module I développed ?

Reply:
it's from [this PR](https://github.com/OCA/stock-logistics-barcode/pull/526)